### PR TITLE
Fixes #150 (Added meshery logo to the readme)

### LINKED ISSUE #150: [Readme] Add meshery logo to the readme
**Current State:**

No meshery logo in readme.

**Desired State:**

Add a meshery logo by taking reference from github.com/meshery/meshery and make it match suitably in dark and light mode.

---
**Contributor Resources**
- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/meshery/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by UtkarshMishra12**
Signed-off-by: Utkarsh Mishra <nasautkarsh@gmail.com>

**Description**
Added meshery logo to the readme

This PR fixes #150 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+<p style="text-align:center;" align="center"><a href="https://layer5.io/meshery"><picture align="center">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/meshery/meshery-logo-dark-text-side.svg"  width="70%" align="center" style="margin-bottom:20px;">
```
Comment (legendarykamal) [inline README.md:2]: please do not use meshery/meshery 's logo, store it locally( if meshsync doesn't have it ) and then use relative referencing to get it.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/meshery/meshery-logo-light-text-side.svg" width="70%" align="center" style="margin-bottom:20px;">
```
Comment (legendarykamal) [inline README.md:3]: here too.

**Review by legendarykamal — CHANGES_REQUESTED** [review]
Changes needed.

File: README.md (hunk 2 above)
Comment (UtkarshMishra12) [reply README.md:3]: On it @legendarykamal

**Comment by UtkarshMishra12** [comment]
@legendarykamal As per your suggestion, I have stored the logos locally and then used relative referencing to render them.
Let me know your thought on it.

**Comment by UtkarshMishra12** [comment]
@legendarykamal I'm closing this PR because of some error on my side and will be opening a new PR for the issue.

**Pull request closed without merging**